### ISSUE TITLE
WebDriver dependency in MVN

### ISSUE THREAD

**Issue by filipcynarski** [FluentLenium/FluentLenium#336]
Now users are required to provide webdrivers dependencies in logs.

It may cause several issues:
- Users forget about that - we need to make logging more descriptive
- Users can use different version of Selenium - we should display Warning message in logs

**Comment by Toilal** [comment]
I think Selenium 3 also take the decision to remove driver dependencies from Selenium 3 project, and user  needs to had them manually. Can you confirm as you tried to use Selenium 3 ?

**Comment by Toilal** [comment]
I postpone this to 3.1 as it depends on #301. We need a global analysis about logging.

**Comment by slawekradzyminski** [comment]
done

**Comment by slawekradzyminski** [comment]
Let’s read Selenium version front dependencies

**Comment by slawekradzyminski** [comment]
In the future, we would probably like to support Gradle as well. Let's wait for #755 and see how important it is.

**Comment by slawekradzyminski** [comment]
Maven part done
